less/toast-demo.less

```
@@ -0,0 +1,27 @@
+// Variables
+@margin: 20px;
+@gap: 10px;
+@font-size: 16px;
```

Instead of pixels, you should be using rem or em units in order to make text resizing be possible. This is especially important for font sizes.

```suggestion
@margin: 1.25rem;
@gap: 0.625rem;
@font-size: 1rem;
```